namesrv/src/main/java/org/apache/rocketmq/namesrv/processor/DefaultRequestProcessor.java

```diff
@@ -635,40 +635,40 @@ private RemotingCommand updateConfig(ChannelHandlerContext ctx, RemotingCommand
         return response;
     }
 
-    private RemotingCommand getConfig(ChannelHandlerContext ctx, RemotingCommand request) {
+  private RemotingCommand getConfig(ChannelHandlerContext ctx, RemotingCommand request) {
         final RemotingCommand response = RemotingCommand.createResponseCommand(null);
 
         String content = this.namesrvController.getConfiguration().getAllConfigsFormatString();
+        if (getResponse(response, content)) {
+            return response;
+        }
+
+        response.setCode(ResponseCode.SUCCESS);
+        response.setRemark(null);
+        return response;
+    }
+
+    private boolean getResponse(RemotingCommand response, String content) {
```

Comment:
Well, returning true/false with the name `getResponse` here seems a bit odd, since it returns true only if there's an exception being caught. How about considering a better method name to make it more readable?

Reply:
> 
Thanks，I will consider a better method name for it!

Reply:
I changed it,you can see it!
 private RemotingCommand getConfig(ChannelHandlerContext ctx, RemotingCommand request) {
        final RemotingCommand response = RemotingCommand.createResponseCommand(null);

        String content = this.namesrvController.getConfiguration().getAllConfigsFormatString();
        setResponse(response, content);
        return response;
    }

    private void setResponse(RemotingCommand response, String content) {
        if (StringUtils.isNotBlank(content)) {
            try {
                response.setBody(content.getBytes(MixAll.DEFAULT_CHARSET));
            } catch (UnsupportedEncodingException e) {
                log.error("getConfig error, ", e);
                response.setCode(ResponseCode.SYSTEM_ERROR);
                response.setRemark("UnsupportedEncodingException " + e);
            }
        }
        response.setCode(ResponseCode.SUCCESS);
        response.setRemark(null);
    }


    private RemotingCommand getClientConfigs(ChannelHandlerContext ctx, RemotingCommand request) {
        final RemotingCommand response = RemotingCommand.createResponseCommand(null);
        final GetRemoteClientConfigBody body = GetRemoteClientConfigBody.decode(request.getBody(), GetRemoteClientConfigBody.class);

        String content = this.namesrvController.getConfiguration().getClientConfigsFormatString(body.getKeys());
        setResponse(response, content);
        return response;
    }